src/components/SignupDirectory.tsx

```diff
@@ -105,7 +105,7 @@ const SignupDirectory: React.FC = () => {
           <SignupDirectoryCard
             hoverable
             onClick={() => {
-              history.push('/signup/pf/1');
+              history.push('/signup/form?pf=true');
```

Comment:
Let's use the enum for this so we can easily change it if needed. I think there's a way to provide URL params as an object 2nd parameter to push(), or you could use a template string?